tools/f18/CMakeLists.txt

```diff
@@ -52,14 +52,14 @@ set(MODULES
 # Create module files directly from the top-level module source directory
 foreach(filename ${MODULES})
   add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/include/${filename}.mod
-    COMMAND f18 -fparse-only -fdebug-semantics ${PROJECT_SOURCE_DIR}/../flang/module/${filename}.f90
+    COMMAND f18 -fparse-only -fdebug-semantics ${CMAKE_CURRENT_SOURCE_DIR}/../../module/${filename}.f90
```

Comment:
I don't understand why we're using hard coded relative paths, but this is an improvement.

We should have a FLANG_SOURCE_DIR variable being set someplace to make this more future-proof.